cli/tests/functional/src/test/java/io/helidon/build/cli/tests/FunctionalUtils.java

```diff
@@ -46,8 +46,9 @@ public class FunctionalUtils {
 
     private static final Logger LOGGER = Logger.getLogger(FunctionalUtils.class.getName());
     private static final String MAVEN_DIST_URL = "https://archive.apache.org/dist/maven/maven-3/%s/binaries/apache-maven-%s-bin.zip";
-
-    static final String ARCHETYPE_URL = String.format("file://%s/cli-data", targetDir(FunctionalUtils.class));
+    static final String ARCHETYPE_URL = OSType.currentOS() == OSType.Windows
+        ? String.format("file:///%s/cli-data", targetDir(FunctionalUtils.class))
```

Comment:
How about using
```java
targetDir(FunctionalUtils.class).resolve("cli-data").toUri().toURL().toString()
```

Would that work ?

`.toUri()` throws a checked exception so that's not convenient, you could use a static initializer or a private static method to catch and wrap into a runtime exception. 

We could also add a method to `FileUtils` like `URI uriOf(Path)`. Then you'd have:

```java
FileUtils.uriOf(targetDir(FunctionalUtils.class).resolve("cli-data")).toURL().toString()
```